bindings/c/fdb_c.cpp

```diff
@@ -333,6 +340,97 @@ extern "C" DLLEXPORT fdb_error_t fdb_future_get_granule_summary_array(FDBFuture*
 	                 *out_count = na.size(););
 }
 
+void setBlobFilePointer(FDBBGFilePointer* dest, const BlobFilePointerRef& source) {
```

Comment:
These helper functions have simple names. Let's declare them in an anonymous namespace